packages/common-test-utils/src/index.ts

```diff
@@ -0,0 +1,129 @@
+import type { CID } from 'multiformats/cid'
```

Comment:
Could this just be a file inside of the `common` package instead of its own new package?  Each new package we introduce adds a noticeable increase in the js-ceramic build time and I'm worried about introducing so many new packages here.

Reply:
Main thing is trying to get test code out of packages that we release. If we're worried about build times, we should switch from npm.

Reply:
One of the things that has happened by having testing utilities in our packages is bleed between packages. Like anchoring related code in common.